src/main/java/lt/verbus/repository/CreditRepository.java

```diff
@@ -0,0 +1,89 @@
+package lt.verbus.repository;
+
+import lt.verbus.exception.EntityNotFoundException;
+import lt.verbus.model.BankAccount;
+import lt.verbus.model.Credit;
+import lt.verbus.model.User;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+public class CreditRepository extends GenericRepository<Credit> {
+
+    public CreditRepository(Connection connection) throws SQLException {
+        super(connection, "credit");
+    }
+
+    public List<Credit> findAll() throws SQLException {
+        return super.findAll();
+    }
```

Comment:
Nereikia tokiu metodu. Gali drasiai is esamu repositor klasiu trinti i viskas veiks (GenericRepository svarbiausia palik).